scripts/support/jlink.sh

```diff
@@ -30,8 +30,8 @@ do_debug() {
     ${GDB} ${TUI} ${ELF_NAME} \
 	-ex "target remote :${GDB_PORT}" \
 	-ex 'monitor halt' \
-	-ex 'load' \
-	-ex 'monitor reset'
+	-ex 'monitor reset' \
+	-ex 'load'
```

Comment:
may also need to update scripts/support/runner/jlink.py

Reply:
Will check

Reply:
Moved change to jlink.py. @mbolivar